index.js

```diff
@@ -46,35 +49,42 @@ function configure() {
 	-----------------------------------
  */
 async function cleanup(url) {
-	console.log(`Fetching: ${url}`);
-	const content = (await got(url)).body;
-
-	console.log('Enhancing web page');
-	const dom = createDom({ url, content });
-
-	/* 
-		Run enhancements
-		----------------
-	*/
-	enhancePage(dom);
-
-	// Run through readability and return
-	const parsed = new Readability(dom.window.document, {
-		classesToPreserve: ['no-href']
-	}).parse();
-
-	return { ...parsed, url };
+	try {
+		spinner.start(`Fetching: ${url}`);
+		const content = (await got(url)).body;
+		spinner.succeed();
+
+		spinner.start('Enhancing web page');
+		const dom = createDom({ url, content });
+
+		/* 
+			Run enhancements
+			----------------
+		*/
+		enhancePage(dom);
+
+		// Run through readability and return
+		const parsed = new Readability(dom.window.document, {
+			classesToPreserve: ['no-href']
+		}).parse();
+
+		spinner.succeed();
+
+		return { ...parsed, url };
+	} catch (error) {
+		spinner.fail(error.message);
+		throw new Error(error);
```

Comment:
Here you can simply throw the original error.
```suggestion
		throw error;
```